### PR TITLE
fix: temporally disable refreshAppend

### PR DESCRIPTION
**Pull request by yirutang**
Will try to add it back in after we sort out the inflight queue stuff.

### CODE REVIEW

**Comment by yirutang** [comment]
Waiting on yayi's approval.

File: google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/StreamWriter.java
Comment (yayi-google) [inline google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/StreamWriter.java:32]: I think it is not recommended.

**Review by yayi-google — APPROVED** [review]

**Review by stephaniewang526 — APPROVED** [review]

**Pull request merged**
